src/manual/Installing/Packages.md

```diff
@@ -107,11 +107,11 @@ Install ClamAV with `zypper`:
 
 > _Note_: openSUSE packages are maintained by Reinhard Max.
 
-### EPEL: Fedora, RHEL, and CentOS
+### EPEL: AlmaLinux and Fedora
 
-EPEL creates ClamAV packages for Fedora (as well as EPEL-enabled CentOS and RHEL). For more information on EPEL, [visit their wiki](https://fedoraproject.org/wiki/EPEL).
+EPEL creates ClamAV packages for Fedora (as well as EPEL-enabled AlmaLinux). For more information on EPEL, [visit their wiki](https://fedoraproject.org/wiki/EPEL).
 
-To enable EPEL for CentOS:
+To enable EPEL for Fedora/AlmaLinux:
 ```bash
 dnf install -y epel-release
```

Comment:
It may be appropriate for RHEL and AlmaLinux (not sure). I just tested with `fedora:latest` and `epel-release` doesn't exist.

Reply:
I didn't test this since I'm not sure how to get RHEL without a license, but I did check their documentation and this step is necessary for both AlmaLinux and RHEL.